schemas/logs/salesforce/insecure-external-assets.yml

```diff
@@ -0,0 +1,145 @@
+schema: Salesforce.InsecureExternalAssets
+version: 0
+referenceURL: https://developer.salesforce.com/docs/atlas.en-us.api.meta/api/sforce_api_objects_eventlogfile_insecureexternalassets.htm
+description: >-
+  Insecure External Assets events contain information about external assets.
+  External assets include images or videos accessed by users over an insecure HTTP protocol.
+  The event lists all your Salesforce pages that contain assets hosted insecurely on third-party sites
+  that users loaded with a Chrome, Firefox, Microsoft Edge, or Safari browser.
+parser:
+  csv:
+    delimiter: ","
+    hasHeader: true
+# Note field sequence is not verified and is likely different.
+fields:
+  - name: EVENT_TYPE
+    type: string
+    required: true
+    validate:
+      allow: ["InsecureExternalAssets"]
+    description: "The type of event. The value is always InsecureExternalAssets."
```

Comment:
nit: I think the `"` should not be needed here? The style we have been following in other yaml files (Cloudformation,  other parsers) is that we put `"` only when needed.

Reply:
Sure, I can change this for uniformity.

Reply:
Done where possible (can't be done only if the string contains some special characters, mainly `:`).